src/Resources/app/storefront/src/js/filter-range.plugin.js

```diff
@@ -0,0 +1,98 @@
+import FilterRangePlugin from 'src/plugin/listing/filter-range.plugin';
+import '../../node_modules/nouislider/distribute/nouislider.css';
+import deepmerge from 'deepmerge';
+import DomAccess from 'src/helper/dom-access.helper';
+
+const noUiSlider = require('@nouislider');
+
+export default class FlFilterRangePlugin extends FilterRangePlugin {
```

Comment:
Selecting a range and then clicking "Reset All" causes the range slider to loose its values:

![image](https://user-images.githubusercontent.com/20555112/109936611-62586700-7cce-11eb-91f7-8c133ebe00d4.png)

![image](https://user-images.githubusercontent.com/20555112/109939683-984b1a80-7cd1-11eb-9e6e-d87b6085a01b.png)

Reply:
Clicking "Reset all" should result in the range slider to reset to its default state.

Removing selected "From price..." should reset the left knob of the range slider.
Removing selected "To price..." should reset the right knob of the range slider.